packages/11ty/_plugins/figures/figure/index.js

```diff
@@ -57,22 +59,33 @@ module.exports = class Figure {
   }
 
   /**
-   * If the `src` is an external resource
-   * @return {Boolean}
+   * Use dimensions of src or first choice as canvas dimensions
```

Comment:
This comment is a little confusing, since it's describing how this getter is intended to be used later rather than what it does directly